disperser/controller/grpcserver/config.go

```diff
@@ -0,0 +1,84 @@
+package grpcserver
+
+import (
+	"fmt"
+	"time"
+)
+
+// Contains configuration for the controller gRPC server
+type Config struct {
+	// Enables the gRPC server
+	//
+	// TODO(litt3): the option to disable the server will be removed once any feature has been fully rolled out which
+	// requires the server.
+	EnableServer bool
+
+	// If true, use the new payment authentication system running on the controller.
+	// If false, payment authentication is disabled and request validation will always fail
+	//
+	// Note: This flag requires EnableServer to be true in order to function.
+	EnablePaymentAuthentication bool
+
+	// Port that the gRPC server listens on
+	GrpcPort string
```

Comment:
Pet peeve, we should always define ports as integers. Many places in the code record them as strings, and I've been changing them as I find them.